Sources/SwiftGenKit/Parsers/Colors/ColorsFileTypeParser.swift

```diff
@@ -24,6 +24,12 @@ protocol ColorsFileTypeParser: AnyObject {
   func parseFile(at path: Path) throws -> Colors.Palette
 }
 
+extension ColorsFileTypeParser {
+  static var allOptions: ParserOptionList {
+    return []
+  }
```

Comment:
I think the default value can be expressed as `static let allOptions: ParserOptionList = []` while still conforming to the protocol? (instead of a computed var)

Reply:
It doesn't.